### PR TITLE
fix(linux): Save QR code in temporary directory

### LINKED ISSUE #5310: bug(linux): shared packages can cause km-config to crash when 'about' clicked
**Describe the bug**

km-config crashes when 'About' is clicked for a package that has been installed for all users, with the following error:

```
Traceback:
 Traceback (most recent call last):
   File "/usr/lib/python3/dist-packages/keyman_config/view_installed.py", line 373, in on_about_clicked
     w = KeyboardDetailsView(self, kmp)
   File "/usr/lib/python3/dist-packages/keyman_config/keyboard_details.py", line 326, in __init__
     img.save(path_qr)
   File "/usr/lib/python3/dist-packages/qrcode/image/pil.py", line 50, in save
     self._img.save(stream, format=format, **kwargs)
   File "/usr/lib/python3/dist-packages/PIL/Image.py", line 2099, in save
     fp = builtins.open(filename, "w+b")
 PermissionError: [Errno 13] Permission denied: '/usr/local/share/keyman/sample/qrcode.png'
```

**To Reproduce**

1. Install a package for all users with:
    ```shell
    sudo km-package-install -s -f build/sample.kmp
    ```
2. Open Keyman Configuration, select the shared package.
3. Click 'About'
4. Crash...

---

**Keyman for Windows/macOS/Linux:**
 - OS: Ubuntu 20.04 
 - Keyman version: 14.0.276

### PR DESCRIPTION
**Pull request by ermshiperete**
Fixes #5310.

### CODE REVIEW

File: linux/keyman-config/keyman_config/keyboard_details.py [hunk 1]
```diff
@@ -323,9 +324,9 @@ def __init__(self, parent, kmp):
         prevlabel = lbl_pad
 
         # If it doesn't exist, generate QR code to share keyboard package
-        path_qr = packageDir + "/qrcode.png"
+        path_qr = path.join(tempfile.gettempdir(), kmp['packageID'] + '_qrcode.png')
```
Comment (darcywong00) [inline linux/keyman-config/keyman_config/keyboard_details.py:327]: Would a temp path be worth adding to #5372?

Comment (mcdurdin) [reply linux/keyman-config/keyman_config/keyboard_details.py:327]: I'd put that into a separate PR if we were going to do it.

**Review by mcdurdin — APPROVED** [review]
LGTM

File: linux/keyman-config/keyman_config/keyboard_details.py (hunk 1 above)
Comment (ermshiperete) [reply linux/keyman-config/keyman_config/keyboard_details.py:327]: At this point I don't see any benefits of doing that - which might change if we use it in more places :smile:

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.76-alpha](https://keyman.com/downloads/releases/alpha/15.0.76)

**Comment by mcdurdin** [comment]
Will we cherry-pick this to stable-14.0?

**Pull request merged**